build.sbt

```diff
@@ -48,10 +48,15 @@ def versionFmt(out: sbtdynver.GitDescribeOutput): String = {
 
 name := "cloudstate"
 
-val GrpcJavaVersion = "1.22.1"
+// Should be in sync with the version used in Akka gRPC
+val GrpcJavaVersion = "1.29.0"
+// Unfortunately we need to downgrade grpc-netty-shaded
+// in the proxy until we have a fix to make it work with
+// native-image
+val GrpcNettyShadedVersion = "1.28.1"
```

Comment:
This doesn't seem to be used anywhere, so grpc-netty-shaded is not downgraded?

Reply:
Oops - I'm fairly sure I just forgot to commit that and it's actually needed for the native-image tck test - but I'll double-check.